IdentityCore/src/webview/embeddedWebview/ui/ios/MSIDWebviewUIController.m

```diff
@@ -58,6 +58,11 @@ + (WKWebViewConfiguration *)defaultWKWebviewConfiguration
 {
     WKWebViewConfiguration *webConfig = [WKWebViewConfiguration new];
 
+    if (@available(iOS 9.0, *))
```

Comment:
Are we still supporting iOS 9?

Reply:
good catch. This was just to follow the starting version that supports this API. the end result should be the same, even though our min supported version is iOS 11.